lighthouse-core/test/audits/dobetterweb/has-inspector-issues-test.js

```diff
@@ -0,0 +1,245 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const InspectorIssuesAudit =
+  require('../../../audits/dobetterweb/has-inspector-issues.js');
+
+/* eslint-env jest */
+
+describe('Has inspector issues audit', () => {
+  let issues;
+  beforeEach(() => {
+    issues = {
+      mixedContent: [],
+      sameSiteCookies: [],
+      blockedByResponse: [],
+      heavyAds: [],
+      contentSecurityPolicy: [],
+    };
+  });
+
+  it('passes when no issues are found', () => {
+    const auditResult = InspectorIssuesAudit.audit({
+      InspectorIssues: issues,
+    });
+    expect(auditResult.score).toBe(1);
+    expect(auditResult.details.items).toHaveLength(0);
+  });
+
+  it('correctly displays mixed content issues', () => {
+    const mixedContentIssues = [
+      {
+        resolutionStatus: 'MixedContentBlocked',
+        insecureURL: 'www.mixedcontent.com',
+        mainResourceURL: 'www.mixedcontent.com',
+      },
+      {
+        resolutionStatus: 'MixedContentWarning',
+        insecureURL: 'www.insecureurl.com',
+        mainResourceURL: 'www.inscureurl.com',
+        request: {
+          requestId: '1',
+          url: 'www.insecureurl.com/request',
+        },
+      },
+    ];
+    issues.mixedContent.push(...mixedContentIssues);
+
+    const auditResult = InspectorIssuesAudit.audit({
+      InspectorIssues: issues,
+    });
+    expect(auditResult.score).toBe(0);
+    expect(auditResult.details.items[0]).toMatchObject({
+      issueType: 'Mixed Content',
+      subItems: {
+        type: 'subitems',
+        items: [
+          {
+            url: 'www.mixedcontent.com',
```

Comment:
maybe leave a comment like
```suggestion
            // Fell back to `mainResourceURL` since no request.
            url: 'www.mixedcontent.com',
```
or something so the subtlety is obvious for future editors of this file?